manual/C/ch_Finance-Quote.xml

```diff
@@ -249,6 +249,201 @@ SYNOPSIS
+      </tip>
+    </sect2>
+
+    <sect2 id="gnc-fq-update">
+      <title>gnc-fq-update</title>
+
+      <abstract>
+        <para>This program installs or updates the &app-fq; software module along with its dependencies.  
+        </para>
+      </abstract>
+
+      <note>
+        <para>This program needs superuser or administrative privileges to succeed in &lin; or &mac; but not in &win;.
+        </para>
+      </note>
+    </sect2>
+
+    <sect2 id="gnc-fq-dump">
+      <title>gnc-fq-dump</title>
+
+      <abstract>
+        <para>This program returns quote data for a source and a list of symbols in a format which is easy
+          to read for humans. It is useful for checking that a given online quote source is online and functional. 
+        </para>
+      </abstract>
+
+      <itemizedlist>
+        <listitem>
+          <para>To get a stock price, run a command from the command line of the form:
+            <cmdsynopsis>
+              <command>gnc-fq-dump</command>
+              <arg choice="opt">-v</arg>
+              <arg choice="plain"><replaceable>Source</replaceable></arg>
+              <arg choice="plain" rep="repeat"><replaceable>Symbol</replaceable></arg>
+            </cmdsynopsis>
+            e.g. <command>gnc-fq-dump -v alphavantage IBM</command>
+          </para>
+        </listitem>
+        
+        <listitem>
+          <para>To retrieve currency exchange rate run the following command:
+            <command>gnc-fq-dump <optional>-v</optional> currency USD EUR</command>
+            <footnote>
+              <para>Since &app-fq; 1.41 the default source for currencies is
+                hardcoded to be <quote>Alpha Vantage</quote>. The old yahoo
+                currencies are still available as <command>gnc-fq-dump yahoo_json USDEUR=X</command>.
+              </para>
+            </footnote>
+          </para>
+        </listitem>
+      </itemizedlist>
+
+      <para>To test that &app-fq; is working for currencies within &app;,
+        <orderedlist numeration="arabic">
+          <listitem>
+            <para>find a two-currency transaction and</para>
+          </listitem>
+          <listitem>
+            <para>right-click on it, then</para>
+          </listitem>
+          <listitem>
+            <para>select &gmi.ac.ed-ex; in the context menu.</para>
+          </listitem>
+        </orderedlist>
+        The <guilabel>Transfer</guilabel> dialog will appear. Click the 
+        <guilabel>Fetch Rate</guilabel> button and if everything is
+        behaving itself the current rate will fill in the exchange rate box.
+      </para>
+    </sect2>
+
+    <sect2 id="gnc-fq-helper">
+      <title>gnc-fq-helper</title>
+
+      <abstract>
+        <para>This program is a filter, which does the same thing as gnc-fq-dump, but in a form which
+          the &app; understands. Input and output are Scheme expressions. This helps developers to decide,
+          if the error is in &app; or &app-fq;.
```

Comment:
Mostly wrong. gnc-fq-helper is the script that GnuCash uses to retrieve quotes. We'll sometimes ask users to run it from the command line instead of gnc-fq-dump because it works a little differently and can show errors that gnc-fq-dump doesn't, but that's not its main purpose. Also this is too technical for users: The usage of 'filter' won't make sense to non-programmers. Last, while the I/O is Scheme for GnuCash 4 it won't be in GnuCash 5.